base/error.go

```diff
@@ -38,6 +39,22 @@ func ErrorAsHTTPStatus(err error) (int, string) {
 	if err == nil {
 		return 200, "OK"
 	}
+
+	switch err.Error() {
+	case gocbcore.ErrKeyNotFound.Error():
+		return http.StatusNotFound, "missing"
+	case gocbcore.ErrKeyExists.Error():
+		return http.StatusConflict, "Conflict"
+	case gocbcore.ErrTimeout.Error():
+		return http.StatusServiceUnavailable, "Database server is over capacity"
```

Comment:
Yeah agreed, that's misleading:

How about this?  It changes that message and adds a little more verbosity for the different "Database server is over capacity" errors

```
	switch err.Error() {
	case gocbcore.ErrKeyNotFound.Error():
		return http.StatusNotFound, "missing"
	case gocbcore.ErrKeyExists.Error():
		return http.StatusConflict, "Conflict"
	case gocbcore.ErrTimeout.Error():
		return http.StatusServiceUnavailable, "Database timeout error (gocbcore.ErrTimeout.Error)"
	case gocbcore.ErrOverload.Error():
		return http.StatusServiceUnavailable, "Database server is over capacity (gocbcore.ErrOverload.Error)"
	case gocbcore.ErrBusy.Error():
		return http.StatusServiceUnavailable, "Database server is over capacity (gocbcore.ErrBusy.Error)"
	case gocbcore.ErrTmpFail.Error():
		return http.StatusServiceUnavailable, "Database server is over capacity (gocbcore.ErrTmpFail.Error)"
	}
```